admin_panel/urls.py

```diff
@@ -19,6 +18,7 @@
     StudentViewset,
     CustomAuthTokenView,
     EmployeeViewset,
+    CourseViewset,
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider consolidating CourseViewset import</b></div>
<div id="fix">

It appears that `CourseViewset` is being imported from both `admin_panel.viewset` and `admin_panel.viewsets`. Consider consolidating these imports to avoid potential confusion and maintenance issues.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -17,6 +17,5 @@
  from admin_panel.viewset import (
      StudentViewset,
      CustomAuthTokenView,
 -    EmployeeViewset,
 -    CourseViewset,
 +    EmployeeViewset,
  )
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/sleep-404/ExamEvoke/pull/3#issuecomment-2499302777>#f8bf08</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged